elastic/kibana
# [Response Ops][Actions] Logging errors from the mustache lambda format functions

Resolves https://github.com/elastic/kibana/issues/173972

## Summary

Changing to log and return error string instead of throwing error inside the mustache format lambda functions. When an error occurs, the lambda will return the error message. This allows the rest of the action message to be rendered, while giving the user some indication an error occurred and logging the error in a way that is discoverable in the logs.

## To verify

Create a rule and add a message like:

```
{{alert.id}} - {{#FormatDate}} {{{context.nope}}}  ; America/New_York ; dddd MMM Do YYYY HH:mm:ss.SSS {{/FormatDate}}
```

When the rule runs and triggers and alert, you should see the alert ID in the notification message along with a `date is empty` message and an error log indicating that the date formatting was unsuccessful due to empty date.
